carsus/io/output/base.py

```diff
@@ -793,13 +878,18 @@ def to_hdf(self, fname):
             f.put('/zeta_data', self.zeta_data.base)
             f.put('/levels', self.levels_prepared)
             f.put('/lines', self.lines_prepared)
-            f.put('/collision_data', self.collisions_prepared)
             f.put('/macro_atom_data', self.macro_atom_prepared)
-            f.put('collision_data_temperatures', 
-                   pd.Series(self.collisions_param['temperatures']))
             f.put('/macro_atom_references',
                   self.macro_atom_references_prepared)
 
+            try:
+                f.put('/collision_data', self.collisions_prepared)
+                f.put('/collision_data_temperatures', 
+                      pd.Series(self.collisions_param['temperatures']))
+
+            except AttributeError:
+                pass
```

Comment:
a comment why an `AttributeError` is fine would be nice.

Reply:
I agree.

Reply:
The comment is already there, are you sure you fetched the changes from September?